v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerShardedMigrationWithMigrationShardIdColumnIT.java

```diff
@@ -247,6 +254,14 @@ public void multiShardMigration() {
             .waitForCondition(createConfig(jobInfo2, Duration.ofMinutes(10)), rowsConditionCheck);
```

Comment:
Isnt this wait for condition being used to specify how long to wait till conditions match? We could probably just bump up these timeouts instead of adding a thread.sleep() for consistency. 


However, seems like we are already giving plenty of wait time.

Reply:
That's only for row count match.
The flakyness is due to CDC propagation delay, for which there's no easy check.